ui/src/components/microsegmentation/RuleRow.js

```diff
@@ -317,6 +319,13 @@ export class RuleRow extends React.Component {
             ''
         );
 
+        let scope = '';
+        if (data && data['conditionsList']) {
+            data['conditionsList'].forEach((item) => {
+                scope = this.stringUtils.getScopeString(item);
+            });
+        }
+
```

Comment:
@chandrasekhar1996 , from what I've noticed all the msd rules only have one item in the `data['conditionsList']`. Are there examples such that there are multiple items in the conditionsList?

Reply:
discussed with chandu on this and found an example with multiple items, will fix code with next commit.

Reply:
fixed.